recipes/ilamb/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "ILAMB" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 15c2e0da0139d406df5883dad428ab8692aeebbb55368db0ed3b93466f9d19db
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python =2
+    - setuptools
+  run:
+    - python =2
+    - cython
+    - basemap >=1.0.7
+    - cf-units >=2.0.0
+    - matplotlib >=1.4.3
+    - mpi4py >=1.3.1
+    - netCDF4 >=1.1.4
+    - numpy >=1.11.0
+    - proj4
+    - scipy >=0.9.0
+    - sympy >=0.7.6
+
+test:
+  imports:
+    - ILAMB
+  commands:
+    - ilamb-doctor -h
+    - ilamb-fetch -h
+    - ilamb-run -h
+    - ilamb-mean -h
+    - ilamb-table -h
+
+about:
+  home: https://bitbucket.org/ncollier/ilamb
+  license: NONE
```

Comment:
Without a license conda-forge is unable to legally distribute this package and, even if we could, nobody could legally use it.

This needs fixing before this can be merged.

Reply:
@chrisburr Ah right. ILAMB has historically been operating like they are just in the public domain, but I forgot you have to explicitly declare that (and that that doesn't work in all jurisdictions). The ILAMB team is working on adopting an OS license, so I'll fix this shortly.

Reply:
@chrisburr I've added the license so we should be g2g.